tests/lib/systemd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Use like systemd_create_and_start_unit(fakestore, "$(which fakestore) -start -dir $top_dir -addr localhost:11028 $@")
+systemd_create_and_start_unit() {
+    printf "[Unit]\nDescription=For testing purposes\n[Service]\nType=simple\nExecStart=%s" "$2" > /run/systemd/system/$1.service
```

Comment:
(nitpick) It might be easier to read (although slightly more verbose) to use:
```
cat >/run/systemd/system/"$1".service <<EOF
[Unit]
Description=For testing purposes

[Service]
Type=simple
ExecStart=$2
EOF
```
But this is not a blocker of course.